langchain/llms/mlflow_ai_gateway.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import annotations
+
+from typing import Any, Dict, List, Mapping, Optional
+
+from pydantic import BaseModel, Extra
+
+from langchain.callbacks.manager import CallbackManagerForLLMRun
+from langchain.llms.base import LLM
+
+
+class Params(BaseModel, extra=Extra.allow):
+    temperature: float = 0.0
+    stop: Optional[List[str]] = None
+    max_tokens: Optional[int] = None
+    candidate_count: int = 1
+
+
+class MlflowAIGateway(LLM):
+    route: str
+    gateway_uri: Optional[str] = None
+    params: Optional[Params] = None
+
+    def __init__(self, **kwargs: Any):
+        import mlflow.gateway
```

Comment:
can we put this in a try/excpet with a nice error message that includes instructions to install mlflow gateway

Reply:
Great idea, I'll make that change!